module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,12 @@
+#include "vowel.hpp"
+#include <algorithm>
```

Comment:
Header includes and standard library includes should be in the another paragraphs